qiskit_experiments/library/randomized_benchmarking/interleaved_rb_experiment.py

```diff
@@ -78,18 +86,50 @@ def __init__(
                            Clifford samples to shorter sequences.
 
         Raises:
-            QiskitError: the interleaved_element is not convertible to Clifford object.
+            QiskitError: if the interleaved_element is invalid:
+                * it has different number of qubits from the qubits argument
+                * it is not convertible to Clifford object
+                * it has an invalid delay (e.g. violating the timing constraints of the backend)
         """
+        # Validations of interleaved_element
+        # - validate number of qubits of interleaved_element
+        if len(qubits) != interleaved_element.num_qubits:
+            raise QiskitError(
+                f"Mismatch in number of qubits between qubits ({len(qubits)})"
+                f" and interleaved element ({interleaved_element.num_qubits})."
+            )
```

Comment:
We could but it would make the specification a bit complicated, something like if `len(qubits) < interleaved_element.num_qubits`, the instruction will be applied to `qubits[:len(qubits)]`. So I prefer to require `len(qubits) == interleaved_element.num_qubits` for simplicity of the spec.

Reply:
ok.